tachio/capabilities/manageprojects.js

```diff
@@ -90,6 +92,7 @@ async function createProject({
       missive_label_id: labelId,
       start_date: startDate || new Date(),
       end_date: endDate,
+      client_email: clientEmail,
```

Comment:
Hi @kyphan1, can you confirm if this will this support a string of multiple email addresses (this is required)?